### ISSUE TITLE
Renaming

### ISSUE THREAD

**Issue by kaermorchen** [lifeart/ember-language-server#366]
Hi @lifeart.

What do you think about renaming the 'Unstable Ember Language Server' to less confusing?

`UELS` is the main VSCode extension for Ember environment. The name `Unstable Ember Language Server` may to confuse new Ember developers. After the renaming we can to remove the [explanations](https://github.com/ember-learn/guides-source/pull/1798) for `Unstable` in the name.

It may be `Power Ember Language Server` or `Epic Ember Language Server` or other what you like.

**Comment by lifeart** [comment]
Renaming brings less clarity for developers who already use UELS, thing may help here, is transform  https://github.com/ember-tooling/vscode-ember to extension pack and mention `lifeart.vscode-ember-unstable` as it's dependency.

See extension pack documentation: https://code.visualstudio.com/blogs/2017/03/07/extension-pack-roundup

It allow developers to install "official extension" and get uELS as it's dependency

**Comment by machty** [comment]
Came here to say this. It honestly couldn't be more confusing, especially when it's presently very difficult to find which plugins/addons/etc are "stable" and future-ready in the Ember ecosystem.

**Comment by patocallaghan** [comment]
Agree, please consider a rename. I constantly have to tell new engineers that this "Unstable" addon is actually the stable one 😅

**Comment by lifeart** [comment]
Let it be `Stable Ember Language Server` (for now)

**Comment by lifeart** [comment]
I'm not closing issue, because may be a case if I need to update vscode namespace and unpublish existing extension. (in a month or such)
